### ISSUE TITLE
`default` theme override breaks footer style 

### ISSUE THREAD

**Issue by cbarrete** [mfontanini/presenterm#66]
It seems that overriding the default section of a theme affects the footer style as well (the issue might generalize to other overrides affecting other style sections).

Reproduction:
```markdown
---
theme:
  override:
    default:
      margin:
        percent: 6
---

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
culpa qui officia deserunt mollit anim id est laborum.
````
Notice that the footer now shows a left-aligned slide count rather than the default progress bar.

**Comment by mfontanini** [comment]
This is a duplicate (see #51) and should be fixed in 0.3.0. Can you double check? I can't reproduce it in master.

**Comment by cbarrete** [comment]
I mistakenly thought I was running it before. I've tried it with your flake and it works as expected. Thanks!
